gradle/dependencies.gradle

```diff
@@ -65,19 +65,18 @@ versions += [
   argparse4j: "0.7.0",
   avro: "1.8.1",
   avroPlugin: "0.10.0",
-  bcpkix: "1.61",
+  bcpkix: "1.62",
   checkstyle: "8.20",
   commonsCli: "1.4",
   gradle: "5.4.1",
   gradleVersionsPlugin: "0.21.0",
   grgit: "3.1.1",
-  httpcomponents: "4.5.7",
-  httpclient: "4.5.8",
```

Comment:
`httpclient` is preent in AK/trunk - https://github.com/apache/kafka/blob/66d81a0e50680916f541703049dd4e911542b5ed/gradle/dependencies.gradle#L72

Best remove it from there if really is unused?

Reply:
It's used there, but not here. I'll reduce the divergence further.